src/main/java/com/playmonumenta/networkrelay/NetworkRelayAPI.java

```diff
@@ -118,11 +118,8 @@ public static Set<String> getOnlineShardNames() throws Exception {
 		return null;
 	}
 
-	private static RabbitMQManager getInstance() throws Exception {
-		@Nullable RabbitMQManager instance = RabbitMQManager.getInstance();
-		if (instance == null) {
-			throw new Exception("NetworkRelay is not connected to rabbitmq");
-		}
+	private static RabbitMQManager getInstance() {
+		RabbitMQManager instance = RabbitMQManager.getInstance();
```

Comment:
throw RuntimeException to guaranty null is not returned?

Reply:
Had the same thought - that runtime exception is thrown in RabbitMQManager.getInstance(), which is why there's no null warning here.